<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabla_posiciones</class>
 <widget class="QDialog" name="tabla_posiciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>158</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de la competencia:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>83</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Competencia1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>63</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Modalidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>67</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Modalidad1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha NÂ°</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>9</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>831</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <column>
    <property name="text">
     <string>Equipo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Puntos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PG</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TF</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DT</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>410</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
